pkg/ebpf/tracer.go

```diff
@@ -166,6 +170,16 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	return pm, nil
 }
 
+// setupExpvars setups up endpoint 8080 as the debug http endpoint and exposes some metrics for tracer
+func (t *Tracer) setupExpvars() {
+	go http.ListenAndServe(":8080", nil)
```

Comment:
should we enable to configure this port via configuration to avoid conflicting options ?